### ISSUE TITLE
[1.9.35] iPhone 6 Plus / 10.0.2, iPhone 5S / 9.3.5 - "Shake to shuffle" functionality is not working as expected.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#849]
Frequency: Every time

Action:
Precondition: "Shake to shuffle" toggle button should be enable.
1: Launch the app.
2: Tap on "START" and select few motifs.
3: Add / skip music and proceed to "Create" screen.
4: Shake the device to shuffle assets.
5: Check if assets are shuffled.

Expected result: Asset's position should shuffle randomly when user shake the device to shuffle assets.

Actual result: 
1: Asset's position is not changing when user shake the device to shuffle the assets.
2: Notification "Shuffle Activated" is displayed at "Preview".

**Comment by xiuie** [comment]
I can kind of reproduce this on iPhone 6/10.0.2.. The shuffle works.. but the thumbnail doesn't get updated.. it's the same bug as the "Can't Add Assets from Create screen"

**Comment by xiuie** [comment]
This is not a duplicate but essentially it will be fixed along with #846 as has the same cause.

**Comment by vysohanych** [comment]
I agree with @xiuie .  The shuffle works but the thumbnail doesn't get updated

**Comment by hiteshbheda** [comment]
@xiuie Shuffle works but assets thumbnails do not update. But if user save lomotif after shuffle and load it again then it's updated

**Comment by testershub** [comment]
@xiuie I can still reproduce this issue for build [1.9.38] on iPhone 6 Plus / 10.0.2.

**Comment by testershub** [comment]
@xiuie I can still reproduce this issue for build [1.9.39] on iPhone 6 Plus / 10.1.1 and iPhone 5S / 9.3.5.
